nix/importTOML.nix

```diff
@@ -0,0 +1,28 @@
+let
+  extraModulesDir = toString ../extra;
+  extraModules = builtins.readDir extraModulesDir;
+in
+file:
+
+{ lib, ... }:
```

Comment:
Bring your own lib, or completely lib-less?

Reply:
This is technically completely lib-less. Since `importTOML` is supposed to return a module, we can just have that module pull `lib` as a module argument. I can clarify that with a comment.

Reply:
Oh yeah, now I see! Yep, a comment might help to guide blind sheep like me around. Roadsigns are always great. :+1:

Reply:
Now I see! Roadsigns are always good :+1: